tools/lib/provision.py

```diff
@@ -199,7 +197,7 @@
     SYSTEM_DEPENDENCIES = [
         *DEBIAN_DEPENDECIES,
         f"postgresql-{POSTGRESQL_VERSION}",
-        f"postgresql-{POSTGRESQL_VERSION}-pgdg-pgroonga",
+        f"postgresql-{POSTGRESQL_VERSION}-pgroonga",
```

Comment:
Should this change be its own commit?  It seems like a change to the Debian setup that is independent of the rest.

Reply:
Done.